rct229/rules/section6/__init__.py

```diff
@@ -7,10 +7,13 @@
     "section6rule3",
     "section6rule4",
     "section6rule6",
+    "section6rule7",
     "section6rule8",
     "section6rule9",
 ]
 
+from rct229.rules.section6 import section6rule7
```

Comment:
Was this being used during development and should not be removed?

Reply:
I don't know why this line was added here, but I really appreciate for pointing this out :).